<launch>
    <!--arg name="camera_link_pose" default="0.247 0.095 0.905 -0.019 0.527 0.016 0.850"/-->
    <arg name="camera_link_pose" default="1.4 0.03 0.58 -0.424 -0.026 0.905 -0.010"/>


    <arg name="ns" default="object_babbling"/>
    <arg name="exec_name" default="controller_node"/>
    <arg name="group_name" default="left_arm"/>

    <arg name="right_electric_gripper" default="false"/>
    <arg name="left_electric_gripper" default="false"/>

    <arg name="left_tip_name" default="left_gripper"/>
    <arg name="right_tip_name" default="right_gripper"/>

    <!--CAFER-->
    <arg name="cafer_type" default="$(arg exec_name)"/>
    <arg name="managed_uuid" default="none"/>
    <!--/CAFER-->

    <node name="statict_transform" pkg="tf" type="static_transform_publisher" args=" $(arg camera_link_pose) /world /camera_link 100"/>

    <rosparam command="load" file="$(find object_babbling)/launch/params/planner_params.yml" ns="/object_babbling/controller_node/planner_parameters"/>

    <node name="$(arg exec_name)" pkg="object_babbling" type="$(arg exec_name)" output="screen">
    <!-- <node name="$(arg exec_name)" pkg="object_babbling" type="$(arg exec_name)" output="screen" launch-prefix="xterm -e gdb -args"> -->
        <rosparam ns="cafer" subst_value="True">
            mgmt: "/management_topic"
            freq: 30.0
            type: "$(arg cafer_type)"
            uuid: "$(arg managed_uuid)"
        </rosparam>
        <rosparam command="load" file="$(find object_babbling)/launch/params/experiment_settings.yml" ns="experiment"/>
    </node>
</launch>
